### DISCUSSION TITLE
Selecting text in the prompt

### DISCUSSION THREAD

**Discussion by metov** [davatorium/rofi#1745] [Q&A]
Does rofi support selecting text in the prompt?

As an example, sometimes I type a string for a program, and then change my mind about what I want to type. Of course I can just dismiss the rofi prompt with Esc, then open it again with a clear prompt. However, from other launchers (like albert) I'm used to being able to select all with Ctrl+A and clearing the text that way. This also makes it possible to select only words with Ctrl+Shift+arrow as well.

Is it possible to do this in rofi? It seems like the input prompt does not support text selection.

**Comment by DaveDavenport** [comment]
There is no selection support in rofi.
But what you describe is just editing, there are  *many* bindings for editing.. (like delete word forward, delete word back, delete till start of line, etc).

see rofi-keys manpage
